Bio/PDB/kdtrees.c

```diff
@@ -0,0 +1,1483 @@
+#include <math.h>
```

Comment:
Can you put an explicit dual license comment header in here? Not sure if we've dual licensed any of the C or C++ code yet, so you may get to set the convention for this.